### PR TITLE
Fail gracefully if there is not an active file just as we fail gracefully if there is no outline.

### PR DESCRIPTION
**Pull request by jacob314**
…

### CODE REVIEW

**Comment by jacob314** [comment]
Fixes https://github.com/flutter/flutter-intellij/issues/4215

File: src/io/flutter/editor/PropertyEditorPanel.java [hunk 1]
```diff
@@ -330,8 +330,7 @@ InspectorObjectGroupManager getGroupManager() {
   int getOffset() {
     if (outline == null) return -1;
     final VirtualFile file = activeFile.getValue();
-
-    assert (activeFile.getValue() != null);
+    if (activeFile == null) return -1;
```
Comment (devoncarew) [inline src/io/flutter/editor/PropertyEditorPanel.java]: We've already dereferenced `activeFile` above; perhaps you meant `file` here? We should also replace the `activeFile.getValue()` reference below with `file`.

Comment (jacob314) [reply src/io/flutter/editor/PropertyEditorPanel.java]: doh. meant to wrote file instead of activeFile. fixed

**Comment by devoncarew** [comment]
(merging)

**Pull request merged**